lib/content-services/src/lib/search/components/search-chip-list/directives/search-chip-list.directive.ts

```diff
@@ -0,0 +1,32 @@
+/*!
+ * @license
+ * Copyright 2019 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Directive, ElementRef, Renderer2 } from '@angular/core';
+
+@Directive({
+  selector: '[adf-a11y-search-chip-list]'
```

Comment:
I see that there are common issues we want to fix throughout the application components, mainly `role` and `tabindex`.  Why not have a configurable generic directive that can do that? what this directive manipulate could not be specific only to search-chip-list or search-chip.